src/components/Select/Select.jsx

```diff
@@ -29,24 +29,38 @@ export default class Select extends React.Component {
     document.addEventListener('click', this.closePopover, true);
 
     // catch the keypress to move the selected items up or down
-    document.addEventListener('keydown', this.keyDownPressed, true);
+    this.selectNode && this.selectNode.addEventListener('keydown', this.keyDownPressed);
   }
 
   componentWillUnmount() {
     document.removeEventListener('click', this.closePopover, true);
-    document.removeEventListener('keydown', this.keyDownPressed, true);
+    this.selectNode && this.selectNode.removeEventListener('keydown', this.keyDownPressed);
   }
 
   keyDownPressed = (e) => {
-    if (e.which === 40) {
-      e.preventDefault();
-      this.onMoveDown();
-    } else if (e.which === 38) {
-      e.preventDefault();
-      this.onMoveUp();
+    const { shortcutsEnabled } = this.props;
+    if (!shortcutsEnabled) return;
+
+    switch (e.which) {
```

Comment:
could we add comments in to explain what key codes these correspond to?

Reply:
Ah, that's a great one @gomezjuliana ! Adding that now 😊 !